src/main/java/it/gov/pagopa/pu/bff/connector/debt_position/DebtPositionServiceImpl.java

```diff
@@ -50,4 +50,10 @@ public PagedModelDebtPosition getDebtPositionByDebtPositionTypeOrgId(Long debtPo
   public boolean deleteDebtPosition(Long debtPositionId, String accessToken) {
     return client.deleteDebtPosition(debtPositionId, accessToken);
   }
+
+  @Override
+  public Long validateOperator(Long debtPositionId, Long organizationId, String operatorExternalUserId, String accessToken) {
```

Comment:
let's hide the conversion between the Long to boolean here
```suggestion
  public Long hasOperatorGrantOnDebtPosition(Long debtPositionId, Long organizationId, String operatorExternalUserId, String accessToken) {
```